packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -46,8 +45,7 @@ export interface MenuItemProps extends VibeComponentProps {
   shouldScrollMenu?: boolean;
   closeMenu?: (option: CloseMenuOption) => void;
   menuRef?: React.RefObject<HTMLElement>;
-  // TODO MenuItem can accept only Menu element as first level child, it accepts MenuChild[] as children even though it is not valid.
-  //  Should be fixed in next major version
+  //// TODO: [breaking] MenuItem can accept only Menu element as first level child, it accepts MenuChild[] as children even though it is not valid
```

Comment:
Prefixed all breaking todos with [breaking] to better find them and also opened tickets